go/packages/dev-mode/agent/forwarder.go

```diff
@@ -113,6 +122,19 @@ func FindRuntimeDone(logs []LogItem) *LogItem {
 	return nil
 }
 
+func StringIncludesInitError(str string) bool {
+	return strings.Contains(str, "\tundefined\tERROR") && strings.Contains(str, "\tundefined\tUncaught Exception")
+}
+
+func FindInitErrorLog(logs []LogItem) *LogItem {
+	for _, log := range logs {
+		if log.LogType == "function" && StringIncludesInitError(log.Record.(string)) {
+			return &log
+		}
+	}
+	return nil
+}
```

Comment:
```suggestion
func stringIncludesInitError(str string) bool {
	return strings.Contains(str, "\tundefined\tERROR") && strings.Contains(str, "\tundefined\tUncaught Exception")
}

func FindInitErrorLog(logs []LogItem) *LogItem {
	for _, log := range logs {
		if log.LogType == "function" && stringIncludesInitError(log.Record.(string)) {
			return &log
		}
	}
	return nil
}
```
`stringIncludesInitError` doesn't need to be exported (public)